MoneyManager/MoneyManager/Repositories/TransactionRepository.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+using MoneyManager.DataAccessLayer.Entities;
+
+namespace MoneyManager.DataAccessLayer.Repositories
+{
+    public class TransactionRepository: Repository<Transaction>
+    {
+        public TransactionRepository(ApplicationContext context) : base(context) { }
+
+        public IEnumerable<Transaction> CurrentMonth(int userId)
+        {
+            DateTime startDate = new DateTime(DateTime.Today.Year, DateTime.Today.Month, 1);
+            DateTime endDate = DateTime.Today;
+            return GetByDateRange(startDate, endDate)
```

Comment:
looks like repeatable pattern
`GetThisMonthItems` or simial method can be added